library/src/main/java/com/eyeem/deviceinfo/Factory.java

```diff
@@ -61,6 +62,7 @@ public static DeviceInfo get(Context base) {
       int smallestWidthDp;
       int statusBarHeight;
       int navigationBarHeight;
+      Point navigationBarAll; // can be width or height
```

Comment:
navigationBar... dimensions, side, length ...or fatness :) ? all is not descriptive at all, also comment with java doc so that anyone using this library will know what this is for